templates/Web/_composition/Flask/Page.Flask.List.AddRoutes.WithCosmos.Mongo/server/server_postaction.py

```diff
@@ -1,14 +1,15 @@
-from flask import Flask, send_from_directory
-from flask import jsonify
-from flask import make_response
+from flask import Flask, jsonify, make_response, send_from_directory
+import os
+from os.path import exists, join
+
 //{[{
-from mongo.mongo_service import *
+from mongo.mongo_service import get, create, destroy
```

Comment:
Talked to @njindal239 about the double new lines here. He will address it in #819